src/components/noticeNoData/NoticeNoLikes.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Typography } from '@mui/material'
+import { makeStyles } from 'tss-react/mui'
+import theme from '../../styles/createTheme'
+
+const NoticeNoLikes = () => {
+  const { classes } = useStyles()
+
+  return (
+    <Typography variant="h2" className={classes.title}>
+      Sorry!
+      <br />
+      There are no likes yet
+    </Typography>
+  )
+}
+
+export default NoticeNoLikes
+
+const useStyles = makeStyles()(() => {
+  return {
+    title: {
+      textAlign: 'center',
+      paddingBottom: '80px',
+      paddingTop: '50px',
+      [theme.breakpoints.down('sm')]: {
+        maxWidth: '290px',
```

Comment:
Could you please explain why do you need this breakpoint with` maxWidth`

Reply:
It's done accordingly the design provided.
<img width="242" alt="image" src="https://github.com/WeFriiends/WeFriiendsWeb/assets/324031/55e99c4c-6288-4b38-a162-4d8c68f5724b">

Reply:
I understood. We have different interpretations of the design. I think that we should have fixed margins on the sides, rather than fixed width of the text.
I think it's better to clarify this point with the designer.

For example for width 550 it looks like: 
![_near-by](https://github.com/WeFriiends/WeFriiendsWeb/assets/81742845/f5bab7c0-f2ef-4671-8add-554c8049a50c)

Reply:
Olga asked to add one more centered container, completed with designer's approvement.